docs/workflows.rst

```diff
@@ -229,3 +229,12 @@ A FreeSurfer subjects directory is created in ``<output dir>/freesurfer``.
 
 A copy of the ``fsaverage`` subject distributed with the running version of
 FreeSurfer is copied into this subjects directory.
+
+
+Susceptibility Distortion Correction (SDC)
+==========================================
```

Comment:
I am skeptical about including SDC documentation without actually fixing SDC. Here's my reasoning: imagine we will merge this now and next week (before you will have a chance to submit a PR fixing SDC) we find a bug and have to release 0.3.2. It will include this documentation misleading users about the inclusion of SDC functionality (when in reality it will still remain broken and unusable).